tests/features/publication.feature

```diff
@@ -48,6 +123,9 @@ Feature: Publication content creation
     And I fill in "Introduction" with "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas felis leo, lobortis non eros in, consequat tempor est. Praesent sit amet sem eleifend, cursus arcu ac, eleifend nunc. Integer et orci sagittis, volutpat felis sit amet, tincidunt amet. Text to remove"
     And I fill in "Subject" with "financing"
     And I fill in "Responsible department" with "European Patent Office"
+    And I fill in "Resource type" with "Acknowledgement receipt"
+    And I fill in "Use existing media" with "Sample image"
+    And I fill in "oe_documents[0][target_id]" with "My Document 1"
```

Comment:
Please use field labels. Refer to comment on previous scenario.

Reply:
I changed the statement to use field labels.